code/modules/uplink/uplink_items.dm

```diff
@@ -1978,23 +1978,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/box/syndie_kit/prisonerbag
 	cost = 2
 
-/datum/uplink_item/device_tools/holo_sight
-	name = "Holographic Sight"
-	desc = "A high-tech holographic sight that improves the aim of the weapon it's attached to."
-	item = /obj/item/attachment/scope/holo
-	cost = 2
-
-/datum/uplink_item/device_tools/vert_grip
-	name = "Vertical Grip"
-	desc = "A vertical foregrip that reduces the shock of firing a weapon. Extra handy for higher recoil guns like the sniper rifle."
-	item = /obj/item/attachment/grip/vertical
-	cost = 2
+/datum/uplink_item/device_tools/attachments
+	name = "Weapon Attachments"
+	desc = "A set of attachments that will make your gun more accurate and stable. \
+			Comes with a laser sight as well, for that extra tactical feel."
+	item = /obj/item/storage/box/syndie_kit/attachments
+	cost = 1
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
-/datum/uplink_item/device_tools/laser_sight
-	name = "Laser Sight"
-	desc = "An aesthetic laser sight that improves your accuracy and shows you where you're aiming."
-	item = /obj/item/attachment/laser_sight
-	cost = 2
+/datum/uplink_item/device_tools/attachments/nukie
+	cost = 10
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```

Comment:
might wanna also return exclude_modes to null, otherwise it'll try to include and exclude the modes at the same time